srml/council/src/lib.rs

```diff
@@ -15,132 +15,86 @@
+//! - **Desired seats:** The number of seats on the council. Can change via governance.
+//! - **Candidacy bond:** Bond required to be a candidate.
+//! - **Voting bond:** Bond required to be permitted to vote.
+//! - **Candidate approval voting call:** Express candidate approval voting is a public call that anyone may execute
 //! by signing and submitting an extrinsic. We ensure that information about the `origin` where the dispatch initiated
 //! is a signed account using `ensure_signed`.
-//!
-//! - **Candidate registration and vote index** Candidate approval votes are only considered before the presentation
-//! period and for candidates that have a registered list slot with an approved candidate index `VoteIndex`.
-//!
-//!  - **Candidate voters security bond (for the first vote)** If it's the voter's first vote and their vote is valid,
-//! then before enacting any operation and changing the storage, a security bond is deducted from them using the
-//! `reserve` function of the Balances module, as it may result in a major alteration of storage. The bond amount
-//! should be sufficient to cover any costs of the substantial execution in case the operation cannot proceed.
-//! The bond is a mitigation measure against the classical blockchain attack scenario since we cannot be certain
-//! that the operation will not require substantial computation. The voters' account id is added to the list `Voters`
-//! of present voters.
-//!
-//! - **Candidate voters' subsequent votes (after their first vote)** If the voter makes a subsequent vote that's
-//! valid, then their vote is recorded in `LastActiveOf`, which maps their account id to the last cleared vote index
-//! that they were active at, and the votes (i.e. yay or nay) for each candidate with a vote index are added to the
-//! `ApprovalsOf` mapping.
-//!
-//! - **Candidate voter inactivity reaping process** After determining the claims' validity, we call `remove_voter`
-//! as follows depending on the claim validity: if the claim was valid delete the inactive voter, otherwise delete
-//! the reporter. Lastly we perform reaping as follows, depending on the claim validity: if the claim is valid call
-//! `repatriate_reserved` to slash the target account of their voter bond and move that value from the targets'
-//! reserved account balance to the free balance of the reporter (beneficiary) and emit a `VoterReaped` event,
-//! otherwise if the claim was invalid call `slash_reserved` to slash the reporter for their bad behaviour in making a
-//! false claim. The reporter's account is slashed by deducting a value from their reserved balance, and by decreasing
-//! the total amount of stake in the system by the amount that was slashed, then we emit a `BadReaperSlashed` event.
+//! - **Reaping process:** Councillors may propose the removal of other, inactive councillors. If the claim is not
+//! valid, the reporter will be slashed.
```

Comment:
Will link to the Staking module, but we should not be repeating definitions. I think for terms like "slash" there should be one place where this is defined to avoid conflicting definitions.

Reply:
Good point